role-service/src/test/java/uk/ac/ox/ndph/mts/role_service/controller/RoleControllerTest.java

```diff
@@ -158,4 +162,24 @@ void whenPostValidPermissionForRole_thenReceiveSuccess() throws Exception {
 
     }
 
+    @Test
+    void whenGetMultipleRolesById_thenReceiveMultipleRoles() throws Exception {
```

Comment:
Can you add a bit more validation to this test so it actually checks the return value? Currently it would pass if the endpoint just returned the ID string passed in.

Reply:
Thanks good point.  I have strengthened this test using jsonpath to check the structure of the returned roles list.  Also confirming that permissions are returned within roles.